# Opencap API localization file.
# Copyright (C) 2024
# This file is distributed under the same license as the PACKAGE opencap-api.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-01-25 11:44-0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: .\mcserver\serializers.py:23
msgid "email-already_exists"
msgstr "The email already exists."

#: .\mcserver\serializers.py:26
msgid "username-already_exists"
msgstr "The username already exists."

#: .\mcserver\templates\admin\set_subject.html:6
msgid "home"
msgstr "Home"

#: .\mcserver\templates\admin\set_subject.html:9
msgid "set_subject"
msgstr "Set Subject"

#: .\mcserver\templates\admin\set_subject.html:14
msgid "set_subject_confirmation"
msgstr "Are you sure you want to set the subject for the next list of sessions?"

#: .\mcserver\templates\email\reset_password_email.html:11
msgid "reset_password_email_greeting"
msgstr "Hi,"

#: .\mcserver\templates\email\reset_password_email.html:12
msgid "reset_password_email_your_username"
msgstr "Your username is:"

#: .\mcserver\templates\email\reset_password_email.html:15
msgid "reset_password_email_click_button"
msgstr "To reset your password click on the following button:"

#: .\mcserver\templates\email\reset_password_email.html:18
msgid "reset_password_email_reset_button_text"
msgstr "Reset my Password"

#: .\mcserver\templates\email\reset_password_email.html:21
msgid "reset_password_email_token"
msgstr "If the button if not working, click on the following token:"

#: .\mcserver\templates\email\reset_password_email.html:23
msgid "reset_password_email_ignore"
msgstr "If you did not request to retrieve your username or change your password, you can ignore this email."

#: .\mcserver\templates\email\reset_password_email.html:26
msgid "reset_password_email_team"
msgstr "OpenCap Team."

#: .\mcserver\templates\email\verification_email.html:11
msgid "verification_code_email_greeting"
msgstr "Hi,"

#: .\mcserver\templates\email\verification_email.html:12
msgid "verification_code_email_your_code"
msgstr "Your verification code is:"

#: .\mcserver\templates\email\verification_email.html:20
msgid "verification_code_email_copy_paste"
msgstr "Please copy-paste it to the OpenCap webapp."

#: .\mcserver\templates\email\verification_email.html:23
msgid "verification_code_email_ignore"
msgstr "If you did not try to log in into OpenCap, you can ignore this email."

#: .\mcserver\templates\email\verification_email.html:26
msgid "verification_code_email_team"
msgstr "OpenCap Team."

#: .\mcserver\views.py:177 .\mcserver\views.py:210 .\mcserver\views.py:267
#: .\mcserver\views.py:306 .\mcserver\views.py:386 .\mcserver\views.py:418
#: .\mcserver\views.py:445 .\mcserver\views.py:515 .\mcserver\views.py:558
#: .\mcserver\views.py:765 .\mcserver\views.py:833 .\mcserver\views.py:855
#: .\mcserver\views.py:885 .\mcserver\views.py:944 .\mcserver\views.py:1023
#: .\mcserver\views.py:1071 .\mcserver\views.py:1141 .\mcserver\views.py:1173
#: .\mcserver\views.py:1242 .\mcserver\views.py:1423 .\mcserver\views.py:1457
#: .\mcserver\views.py:1481 .\mcserver\views.py:1509 .\mcserver\views.py:1604
#: .\mcserver\views.py:1632 .\mcserver\views.py:1659 .\mcserver\views.py:1692
#: .\mcserver\views.py:1716
msgid "undefined_uuid"
msgstr "Undefined UUID."

#: .\mcserver\views.py:190 .\mcserver\views.py:194 .\mcserver\views.py:198
#: .\mcserver\views.py:244 .\mcserver\views.py:248 .\mcserver\views.py:252
#: .\mcserver\views.py:278 .\mcserver\views.py:282 .\mcserver\views.py:286
#: .\mcserver\views.py:290 .\mcserver\views.py:294 .\mcserver\views.py:314
#: .\mcserver\views.py:323 .\mcserver\views.py:327 .\mcserver\views.py:331
#: .\mcserver\views.py:373 .\mcserver\views.py:377 .\mcserver\views.py:393
#: .\mcserver\views.py:397 .\mcserver\views.py:401 .\mcserver\views.py:405
#: .\mcserver\views.py:409 .\mcserver\views.py:428 .\mcserver\views.py:432
#: .\mcserver\views.py:436 .\mcserver\views.py:455 .\mcserver\views.py:459
#: .\mcserver\views.py:463 .\mcserver\views.py:505 .\mcserver\views.py:546
#: .\mcserver\views.py:571 .\mcserver\views.py:600 .\mcserver\views.py:604
#: .\mcserver\views.py:608 .\mcserver\views.py:612 .\mcserver\views.py:794
#: .\mcserver\views.py:798 .\mcserver\views.py:802 .\mcserver\views.py:820
#: .\mcserver\views.py:846 .\mcserver\views.py:868 .\mcserver\views.py:872
#: .\mcserver\views.py:876 .\mcserver\views.py:918 .\mcserver\views.py:922
#: .\mcserver\views.py:926 .\mcserver\views.py:930 .\mcserver\views.py:934
#: .\mcserver\views.py:1006 .\mcserver\views.py:1010 .\mcserver\views.py:1028
#: .\mcserver\views.py:1032 .\mcserver\views.py:1036 .\mcserver\views.py:1044
#: .\mcserver\views.py:1048 .\mcserver\views.py:1057 .\mcserver\views.py:1120
#: .\mcserver\views.py:1124 .\mcserver\views.py:1156 .\mcserver\views.py:1160
#: .\mcserver\views.py:1217 .\mcserver\views.py:1221 .\mcserver\views.py:1225
#: .\mcserver\views.py:1229 .\mcserver\views.py:1233 .\mcserver\views.py:1292
#: .\mcserver\views.py:1296 .\mcserver\views.py:1300 .\mcserver\views.py:1304
#: .\mcserver\views.py:1308 .\mcserver\views.py:1396 .\mcserver\views.py:1437
#: .\mcserver\views.py:1441 .\mcserver\views.py:1445 .\mcserver\views.py:1464
#: .\mcserver\views.py:1468 .\mcserver\views.py:1472 .\mcserver\views.py:1492
#: .\mcserver\views.py:1496 .\mcserver\views.py:1500 .\mcserver\views.py:1520
#: .\mcserver\views.py:1524 .\mcserver\views.py:1528 .\mcserver\views.py:1615
#: .\mcserver\views.py:1619 .\mcserver\views.py:1623 .\mcserver\views.py:1642
#: .\mcserver\views.py:1646 .\mcserver\views.py:1650 .\mcserver\views.py:1671
#: .\mcserver\views.py:1675 .\mcserver\views.py:1679 .\mcserver\views.py:1699
#: .\mcserver\views.py:1703 .\mcserver\views.py:1707 .\mcserver\views.py:1723
#: .\mcserver\views.py:1727 .\mcserver\views.py:1731 .\mcserver\views.py:1739
#: .\mcserver\views.py:1777 .\mcserver\views.py:1805 .\mcserver\views.py:1823
#: .\mcserver\views.py:1844 .\mcserver\views.py:1908 .\mcserver\views.py:1912
#: .\mcserver\views.py:1963 .\mcserver\views.py:1967 .\mcserver\views.py:2002
#: .\mcserver\views.py:2016 .\mcserver\views.py:2020 .\mcserver\views.py:2043
#: .\mcserver\views.py:2048
msgid "error"
msgstr "Error: %(error_message)s"

#: .\mcserver\views.py:191 .\mcserver\views.py:245 .\mcserver\views.py:279
#: .\mcserver\views.py:315 .\mcserver\views.py:394 .\mcserver\views.py:429
#: .\mcserver\views.py:456 .\mcserver\views.py:572 .\mcserver\views.py:605
#: .\mcserver\views.py:795 .\mcserver\views.py:869 .\mcserver\views.py:919
#: .\mcserver\views.py:1007 .\mcserver\views.py:1029 .\mcserver\views.py:1121
#: .\mcserver\views.py:1157 .\mcserver\views.py:1218 .\mcserver\views.py:1293
msgid "session_uuid_not_found"
msgstr "Sorry, we couldn't find a session with UUID: %(uuid)s"

#: .\mcserver\views.py:195 .\mcserver\views.py:249 .\mcserver\views.py:291
#: .\mcserver\views.py:328 .\mcserver\views.py:406 .\mcserver\views.py:433
#: .\mcserver\views.py:460 .\mcserver\views.py:609 .\mcserver\views.py:625
#: .\mcserver\views.py:799 .\mcserver\views.py:873 .\mcserver\views.py:931
#: .\mcserver\views.py:1011 .\mcserver\views.py:1033 .\mcserver\views.py:1125
#: .\mcserver\views.py:1161 .\mcserver\views.py:1230 .\mcserver\views.py:1305
msgid "session_uuid_not_valid"
msgstr "Sorry, the session UUID: %(uuid)s is not valid"

#: .\mcserver\views.py:199 .\mcserver\views.py:253
msgid "calibration_error"
msgstr "There was an error during the calibration step. Please try again."

#: .\mcserver\views.py:283 .\mcserver\views.py:319 .\mcserver\views.py:398
#: .\mcserver\views.py:923 .\mcserver\views.py:1222 .\mcserver\views.py:1297
msgid "login_needed"
msgstr "Sorry, you have to log in to access to this session."

#: .\mcserver\views.py:287 .\mcserver\views.py:324 .\mcserver\views.py:402
#: .\mcserver\views.py:838 .\mcserver\views.py:927 .\mcserver\views.py:1226
#: .\mcserver\views.py:1301 .\mcserver\views.py:1567
msgid "permission_denied"
msgstr "You do not have permissions to access this data."

#: .\mcserver\views.py:295 .\mcserver\views.py:332
msgid "session_retrieve_error"
msgstr "There was an error while retrieving the session. Please try again."

#: .\mcserver\views.py:374 .\mcserver\views.py:1045 .\mcserver\views.py:1616
#: .\mcserver\views.py:1643 .\mcserver\views.py:1672 .\mcserver\views.py:1700
#: .\mcserver\views.py:1724
msgid "subject_uuid_not_found"
msgstr "Sorry, we couldn't find a subject with UUID: %(uuid)s"

#: .\mcserver\views.py:378
msgid "session_not_valid"
msgstr "There was an error while checking the validity of the session. Please try again."

#: .\mcserver\views.py:410
msgid "session_permanent_remove_error"
msgstr "There was an error while permanently removing the session. Please try again."

#: .\mcserver\views.py:437
msgid "session_remove_error"
msgstr "There was an error while removing the session. Please try again."

#: .\mcserver\views.py:464
msgid "session_restore_error"
msgstr "There was an error while restoring the session. Please try again."

#: .\mcserver\views.py:506 .\mcserver\views.py:613
msgid "session_create_error"
msgstr "There was an error while creating the session. Please try again."

#: .\mcserver\views.py:547
msgid "qr_retrieve_error"
msgstr "There was an error while retrieving the session. Please try again."

#: .\mcserver\views.py:601
msgid "user_not_found"
msgstr "Sorry, we couldn't find the user."

#: .\mcserver\views.py:803
msgid "trial_record_error"
msgstr "There was an error while recording the trial. Please try again."

#: .\mcserver\views.py:821 .\mcserver\views.py:847
msgid "session_download_error"
msgstr "There was an error while downloading the session. Please try again."

#: .\mcserver\views.py:877 .\mcserver\views.py:935
msgid "session_get_settings_error"
msgstr "There was an error while getting the settings of the session. Please try again."

#: .\mcserver\views.py:1015
msgid "session_set_metadata_error"
msgstr "There was an error while setting the metadata of the session. Please try again."

#: .\mcserver\views.py:1037 .\mcserver\views.py:1049 .\mcserver\views.py:1058
msgid "subject_assign_error"
msgstr "There was an error while assigning this subject to the session. Please try again."

#: .\mcserver\views.py:1129 .\mcserver\views.py:1165
msgid "trial_cancel_error"
msgstr "There was an error while canceling the trial. Please try again."

#: .\mcserver\views.py:1234
msgid "calibration_image_retrieve_error"
msgstr "There was an error while retrieving the calibration image. Please try again."

#: .\mcserver\views.py:1309
msgid "neutral_image_retrieve_error"
msgstr "There was an error while retrieving the neutral image. Please try again."

#: .\mcserver\views.py:1397
msgid "trial_dequeue_error"
msgstr "There was an error while dequeuing the trials. Please try again."

#: .\mcserver\views.py:1438 .\mcserver\views.py:1465 .\mcserver\views.py:1493
#: .\mcserver\views.py:1521
msgid "trial_uuid_not_found"
msgstr "Sorry, we couldn't find a trial with UUID: %(uuid)s."

#: .\mcserver\views.py:1442 .\mcserver\views.py:1469 .\mcserver\views.py:1497
#: .\mcserver\views.py:1525
msgid "trial_uuid_not_valid"
msgstr "Sorry, the trial UUID: %(uuid)s is not valid."

#: .\mcserver\views.py:1446
msgid "trial_rename_error"
msgstr "There was an error while renaming the trial. Please try again."

#: .\mcserver\views.py:1473
msgid "trial_permanent_remove_error"
msgstr "There was an error while permanently removing the trial. Please try again."

#: .\mcserver\views.py:1501
msgid "trial_remove_error"
msgstr "There was an error while removing the trial. Please try again."

#: .\mcserver\views.py:1529
msgid "trial_restore_error"
msgstr "There was an error while removing the trial. Please try again."

#: .\mcserver\views.py:1620 .\mcserver\views.py:1647 .\mcserver\views.py:1676
#: .\mcserver\views.py:1704 .\mcserver\views.py:1728
msgid "subject_uuid_not_valid"
msgstr "Sorry, the subject UUID: %(uuid)s is not valid"

#: .\mcserver\views.py:1624
msgid "subject_remove_error"
msgstr "There was an error while removing the subject. Please try again."

#: .\mcserver\views.py:1651
msgid "subject_restore_error"
msgstr "There was an error while restoring the subject. Please try again."

#: .\mcserver\views.py:1680 .\mcserver\views.py:1708 .\mcserver\views.py:1740
msgid "subject_create_error"
msgstr "There was an error while creating a new subject. Please try again."

#: .\mcserver\views.py:1732
msgid "subject_permanent_remove_error"
msgstr "There was an error while permanently removing the subject. Please try again."

#: .\mcserver\views.py:1778
msgid "user_create_error"
msgstr "There was an error while creating your user. Please try again."

#: .\mcserver\views.py:1791 .\mcserver\views.py:1806
msgid "user_delete_error"
msgstr "There was an error while deleting your user. Please try again."

#: .\mcserver\views.py:1798
msgid "user_removed"
msgstr "The user was removed."

#: .\mcserver\views.py:1800
msgid "user_not_authenticated"
msgstr "Sorry, the user is not authenticated."

#: .\mcserver\views.py:1802
msgid "confirmation_username_not_correct"
msgstr "Looks like the verification code is incorrect. Please double check and try again."

#: .\mcserver\views.py:1824 .\mcserver\views.py:1845
msgid "user_update_error"
msgstr "There was an error while updating your user. Please try again."

#: .\mcserver\views.py:1909
msgid "credentials_incorrect"
msgstr "Looks like the username or password you entered is incorrect. Please double check and try again."

#: .\mcserver\views.py:1913
msgid "login_error"
msgstr "There was an error while logging in. Please try again."

#: .\mcserver\views.py:1949
msgid "reset_password_email_subject"
msgstr "Opencap - Forgot Username or Password"

#: .\mcserver\views.py:1964
msgid "account_email_not_found"
msgstr "Sorry, we couldn't find an account associated with that email."

#: .\mcserver\views.py:1968
msgid "error_reset_password"
msgstr "There was an error while resetting your password. Please try again."

#: .\mcserver\views.py:2003 .\mcserver\views.py:2017
msgid "reset_password_link_expired"
msgstr "The link to reset your password has expired or does not exist. Try resetting your password again."

#: .\mcserver\views.py:2021
msgid "new_password_creation_error"
msgstr "There was an error while creating your new password. Please try again."

#: .\mcserver\views.py:2044
msgid "verification_error"
msgstr "There was an error during the verification process. Please try again."

#: .\mcserver\views.py:2049
msgid "verification_code_incorrect"
msgstr "Looks like the verification code is incorrect. Please double check and try again."
